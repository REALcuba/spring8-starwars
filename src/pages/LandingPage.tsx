import Header from '../components/header/Header';
import Nav from '../components/nav/Nav';

const LandingPage= () => {

    return (
        <section className='h-screen'>
            <Header />
            <Nav />
            {/* <Nav/> */}
            {/* <button className='bg-blue-500 rounded-2xl p-2' onClick={handleLogin}>Login</button> */}
            <h1>Esta es la landing</h1>
        </section>
    )
}

export default LandingPage