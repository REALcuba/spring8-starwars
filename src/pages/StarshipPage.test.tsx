
import LandingPage from "./LandingPage";
import { describe, it, expect } from "vitest";



//testing component LandingPage that include the test: Esta es la landing

describe("LandingPage", () => {
    it("should be a function", () => {
        expect(typeof LandingPage).toBe('function');
    });
    it("should render correctly", () => {
        expect(LandingPage).toMatchSnapshot();
    });
    it('body text should match Esta es la landing', () => {
        expect('Esta es la landing').toMatchSnapshot('Esta es la landing');
    })
})




