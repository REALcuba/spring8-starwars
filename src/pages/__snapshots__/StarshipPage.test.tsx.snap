// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LandingPage > body text should match Esta es la landing > Esta es la landing 1`] = `"Esta es la landing"`;

exports[`LandingPage > should match Esta es la landing > Esta es la landing 1`] = `"Esta es la landing"`;

exports[`LandingPage > should render correctly 1`] = `[Function]`;
